File: golang/2011. 执行操作后的变量值.go
func finalValueAfterOperations(operations []string) int {
    ans:=0
    for _,x:=range operations{
        if(x[0]=='+'||x[2]=='+') {
            ans++
        }
        if(x[0]=='-'||x[2]=='-') {
            ans--
        }
    }
    return ans
}

File: 刷题/cpp/2011. 执行操作后的变量值.cpp
class Solution {
public:
    int finalValueAfterOperations(vector<string>& operations) {
        int ans=0;
        for(auto &s:operations){
            if(s[0]=='+'||s[2]=='+') ans++;
            if(s[0]=='-'||s[2]=='-') ans--;
        }
        return ans;
    }
};